substrate/frame/revive/src/exec.rs

```diff
@@ -1265,6 +1273,16 @@ where
 	fn account_balance(&self, who: &T::AccountId) -> U256 {
 		T::Currency::reducible_balance(who, Preservation::Preserve, Fortitude::Polite).into()
 	}
+
+	/// Replaces the `last_frame_output` of the **caller** frame with the default value.
+	fn take_caller_output(&mut self) {
```

Comment:
Do we really need that? It's only called from one place. It's also not really taking. Just dropping.

Reply:
No, you are very right. This helper method shouldn't exist, because dropping the caller output should happen in only a single place anyways. Removed it.